src/util/rocksdb/mod.rs

```diff
@@ -134,11 +135,27 @@ fn check_and_open(
     let existed: Vec<&str> = cfs_list.iter().map(|v| v.as_str()).collect();
     let needed: Vec<&str> = cfs_opts.iter().map(|x| x.cf).collect();
 
+    // Turn "dynamic level size" off for existing column family which was off before.
+    // column families are small, HashMap isn't necessary
+    let (_, cf_decs) = load_latest_options(path, &Env::default(), false)?;
+    let tune_dynamic_level_size =
```

Comment:
I think we'd better extract a function to check options compatibility, since we may check other options in the future.